### PR TITLE
fix(content-releases): update options to match designs

### PR DESCRIPTION
**Pull request by markkaylor**
### What does it do?

- Update the options in the select to match the designs
- Add translations for each option

### How to test it?

- The option list should match the design, when an option is selected it should match the option label. The groupings should work as they did before.

CS-519

### CODE REVIEW

File: packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx [hunk 1]
```diff
@@ -327,24 +327,35 @@ export const ReleaseDetailsLayout = ({
...
  * ReleaseDetailsBody
  * -----------------------------------------------------------------------------------------------*/
+const GROUP_BY_OPTIONS = ['contentType', 'locale', 'action'] as const;
+const getGroupByOptionLabel = (value: (typeof GROUP_BY_OPTIONS)[number]) => {
+  switch (value) {
+    case 'contentType':
+      return {
+        id: 'content-releases.pages.ReleaseDetails.groupBy.option.content-type',
+        defaultMessage: 'Content-Types',
+      };
```
Comment (simotae14) [inline packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx:340]: maybe we can reduce a little bit the code using the default for this case and remove this part

File: packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx [hunk 2]
```diff
@@ -357,7 +368,9 @@ const ReleaseDetailsBody = () => {
     isError: isReleaseError,
     error: releaseError,
   } = useGetReleaseQuery({ id: releaseId });
+
   const release = releaseData?.data;
+  const selectedGroupBy = query?.groupBy || 'contentType';
```
Comment (simotae14) [inline packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx:373]: this var right now it any, can we explicit the type as 
`const selectedGroupBy: (typeof GROUP_BY_OPTIONS)[number] = query?.groupBy || 'contentType';`

**Review by simotae14 — COMMENTED** [review]
Overall it looks good and it works, there are just two small comments

File: packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx (hunk 2 above)
Comment (markkaylor) [reply packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx:373]: The inferred type seems to be coming through as `ReleaseActionGroupBy` on my end 🤔 
Screenshot 2024-01-08 at 17 15 44

Comment (simotae14) [reply packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx:373]: ok, for me was an any but if you have already the inferred type that's ok for me

**Review by simotae14 — APPROVED** [review]
👍🏽

**Pull request merged**